Formula/elektra.rb

```diff
@@ -17,7 +17,9 @@ class Elektra < Formula
   def install
     mkdir "build" do
       system "cmake", "..", "-DBINDINGS=cpp", "-DTOOLS=kdb;",
-                            "-DPLUGINS=NODEP", *std_cmake_args
+                            "-DPLUGINS=NODEP",
+                            "-DCMAKE_C_COMPILER=/usr/bin/clang",
```

Comment:
checkout this PR (#55230) for the honebrew shims fix.

Reply:
Thank you for the review. I changed the formula accordingly. Now it uses `inreplace` to fix the shims path. Hope that works for you.